lib/jekyll.rb

```diff
@@ -153,8 +153,9 @@ def sites
     def sanitized_path(base_directory, questionable_path)
       return base_directory if base_directory.eql?(questionable_path)
 
+      questionable_path.insert(0, '/') if questionable_path.start_with?('~')
```

Comment:
@mastahyeti Hello! Could I borrow your :eyes: for a quick moment? We're seeing issues with questionable paths which start with `~`, such as `~$microsoftSwp.docx`. This is an attempt to fix the `ArgumentError`. Is it safe? :lock: 

```irb
>> File.expand_path('~$microsoftSwp.docx', '/')
ArgumentError: user $microsoftSwp.docx doesn't exist
	from (irb):1:in `expand_path'
	from (irb):1
	from /Users/parkr/.rbenv/versions/2.3.0/bin/irb:11:in `<main>'
>> File.expand_path('/~$microsoftSwp.docx', '/')
=> "/~$microsoftSwp.docx"
```

Figured I'd ask you before merging in Jekyll so we can get some extra reassurances. :heart: